<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton_Search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTableView" name="Table"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="lineSearchReq">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_Reset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Help">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_Sort">
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton_AddNew">
      <property name="text">
       <string>Add New</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_table"/>
    <addaction name="actionSave_table_without_filter"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_table">
   <property name="text">
    <string>Save table from screen</string>
   </property>
  </action>
  <action name="actionSave_table_without_filter">
   <property name="text">
    <string>Save table without filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
